.github/workflows/release_candidate.yml

```diff
@@ -0,0 +1,50 @@
+name: release candidate
+
+on:
+  push:
+    branches:
+      - release-please-*
+      - mxyng/release-please-build
+
+jobs:
+  candidate-name:
+    runs-on: ubuntu-latest
+    outputs:
+      candidate_name: ${{ steps.candidate-name.outputs.candidate_name }}
+    steps:
+      - uses: actions/checkout@v3
+      - id: candidate-name
+        run: |
+          apt-get update && apt-get install -y jq
+          CANDIDATE_NAME=$(jq -r '.["."]' <.release-please-manifest.json)-rc
+          echo "::set-output name=candidate_name::$CANDIDATE_NAME"
+
+  candidate-build:
+    needs: [candidate-name]
+    uses: ./.github/workflows/build.yml
+    with:
+      RELEASE_NAME: ${{ needs.candidate-name.outputs.candidate_name }}
+    secrets: inherit
+
+  candidate-upload:
+    needs: [candidate-name, candidate-build]
+    uses: ./.github/workflows/upload.yml
+    with:
+      RELEASE_NAME: ${{ needs.candidate-name.outputs.candidate_name }}
+      ENVIRONMENT: development
+      NIGHTLY: true
+    secrets: inherit
+
+  restart-services:
+    runs-on: ubuntu-latest
+    needs: [candidate-upload]
+    environment: development
+    steps:
+      - run: brew install infrahq/tap/infra
+      - run: |
+          infra login ${{ secrets.INFRA_SERVER }} --key ${{ secrets.INFRA_ACCESS_KEY }}
+          infra use ${{ secrets.INFRA_DESTINATION }}
+      - uses: azure/setup-kubectl@v2.0
+      - run: |
```

Comment:
This doesn't work at the moment due to #2310

Reply:
Should we leave this part out for now until #2310 is fixed?

Reply:
For sure. This change exists as a separate commit so it's trivial to pull out